pyphare/pyphare/core/box.py

```diff
@@ -0,0 +1,119 @@
+
+
+
+class Box:
+    """represents a box in AMR index cell
+    lower, upper : lower and upper cell AMR indexes
+    """
+
+    def __init__(self, lower, upper):
+        self.lower = lower
+        self.upper = upper
+
+    def __mul__(self, box2):
+        """
+        return the box resulting from the intersection
+        of box2 with self. No intersection returns None
+        """
+        box1 = self
+        inter = Box(max(box1.lower, box2.lower), min(box1.upper, box2.upper))
+        if inter.upper < inter.lower :
```

Comment:
cosmetic

inverse the if, drop the else 
none is default return

Reply:
done